architecture/routing.adoc

```diff
@@ -112,10 +162,25 @@ Passthrough termination is configured by setting `tls:termination` to `passthrou
 }
 ----
 
-=== Re-encryption Termination
-Re-encryption is a type of edge termination where the client encrypts communication with a certificate, which is then re-encrypted with a different certificate when the traffic reaches the destination; with the router using health checks to determine the authenticity of the host.
+<1> Set the `*termination*` field to `passthrough`.
```

Comment:
Maybe this is the render that I'm looking at, but i don't think there's a <1> in the example block. Like, here's a footnote starting with <1>, but it's pointing nowhere. I suggest there be a <1> thrown in in line 161.